### ISSUE TITLE
Loss gets stuck

### ISSUE THREAD

**Issue by javierrodenas** [NVlabs/Deep_Object_Pose#193]
Hello, 

I am experimenting the following issue:

![image](https://user-images.githubusercontent.com/50680821/143269318-5791f7bf-547c-49dd-81ed-4b48c6893375.png)

It is getting stuck around 0.006-0.005. Does it happen to you? 

Thank you in advance

**Comment by TontonTremblay** [comment]
I am not sure, what sort of GPU do you have? Which data are you using?

**Comment by javierrodenas** [comment]
I am using the 2080 RTX Ti (12 GB).

On the other hand, I am using this data currently  (some examples):
![009060](https://user-images.githubusercontent.com/50680821/143407473-6f5080cd-6186-4e2e-9942-c02e15527eab.png)
![009062](https://user-images.githubusercontent.com/50680821/143407513-1f823f05-3ce3-42b3-abd1-bf6cc804e95e.png)

My experience with this environment is that it does not matter which data I use, the loss is always around 0.005

**Comment by javierrodenas** [comment]
I was testing different parameters with same type of data:

Green: learning_rate 0.0001 and batch_size 16
Blue: learning_rate 0.001 and batch_size 8

![image](https://user-images.githubusercontent.com/50680821/143582896-9d12dc6f-68b1-4091-9c2a-749f207e3f29.png)


The result of train loss is more of less the same and it gets stuck around 0.005.

Any recommendation? Do you experimented the same?

Thank you in advance!

**Comment by TontonTremblay** [comment]
It might be caused by the slightly symmetrical object.

**Comment by javierrodenas** [comment]
I can try with non symetrical object.

Another possible issue here could be the color. I comment that because in another github issue I saw a guy who was playing with different color cubes #128. Could my color be an issue? It is white/light grey.

I removed the noise and the loss got to 0.003 (It decreased from 0.005 to 0.003) but it is not able to detect the piece during the predictions

**Comment by TontonTremblay** [comment]
can you share how the belief maps look like?
